# Desired Goal pose
string motor_group
bool only_pos
geometry_msgs/Point position
geometry_msgs/Point orientation
---
bool result